bacon/controller/CoreLogic.swift

```diff
@@ -49,6 +50,25 @@ class CoreLogic: CoreLogicInterface {
             description=\(description) location=\(location as Optional).
             """)
         try transactionManager.saveTransaction(currentTransaction)
+        guard let prediction = prediction else {
+            // No prediction was used, hence no need to check whether prediction is accepted
+            return
+        }
+        if doAcceptPrediction(transaction: currentTransaction, prediction: prediction) {
```

Comment:
Should be `doesAcceptPrediction` if you want to follow Swift api naming guidelines